### ISSUE TITLE
Application crash

### ISSUE THREAD

**Issue by JerrySB03** [IHtDzenda/shalina-tui#27]
Exception:
```
Unhandled exception. System.AggregateException: One or more errors occurred. (An item with the same key has already been added. Key: 134_2453_240902)
 ---> System.ArgumentException: An item with the same key has already been added. Key: 134_2453_240902
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at Core.Api.Maps.Prague.PidLiveData.getTransports(ValueTuple`2 boundingBox, Config config) in /home/jare/Programming/C#/shalina-tui/api/maps/Prague/prague-transport.cs:line 107
   at Core.Api.Maps.TransportInterface.getData(ValueTuple`2 boundingBox, Config config, Boolean useCache) in /home/jare/Programming/C#/shalina-tui/api/maps/transport.cs:line 52
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.get_Result()
   at Core.Api.Maps.TransportInterface.CacheThread(Config config) in /home/jare/Programming/C#/shalina-tui/api/maps/transport.cs:line 40
   at Core.Api.Maps.TransportInterface.<>c__DisplayClass4_0.<getData>b__0() in /home/jare/Programming/C#/shalina-tui/api/maps/transport.cs:line 58
```

**Comment by IHtDzenda** [comment]
Do you know what is the root of this issue ?

**Comment by JerrySB03** [comment]
fixed in #31
